docs/setup/settings.asciidoc

```diff
@@ -253,6 +253,10 @@ running behind a proxy. Use the `server.rewriteBasePath` setting to tell Kibana
 if it should remove the basePath from requests it receives, and to prevent a
 deprecation warning at startup. This setting cannot end in a slash (`/`).
 
+[[server-cors]]`server.cors:`:: *Default: `false`* Set to `true` to enable CORS support, this setting is required in order to configure `server.cors.origin`.
+
+`server.cors.origin:`:: *Default: none* Used to specify origins, "origin" must be an array. This setting requires `server.cors: true` set. Use `server.cors.origin: ["*"]` to accept all origins.
```

Comment:
I'm actually not sure the `server.cors.origins` feature is implemented in production mode. We still have an open issue about this: https://github.com/elastic/kibana/issues/16714

Someone will need to verify, I may have some time later today.